brayns/common/scene/Scene.cpp

```diff
@@ -244,19 +240,20 @@ void Scene::buildDefault( )
 
     // Lamp
     ++material;
+    const float y = 0.99f;
```

Comment:
Because the value is used several times in the following lines. To make it coherent, I defined the x and z values in the same way.